src/Place/Pages/PlacePageNavigator/PlacePageNavigator.tsx

```diff
@@ -0,0 +1,32 @@
+import { createNativeStackNavigator, NativeStackScreenProps } from '@react-navigation/native-stack';
+
+import { PlacesPropsInterface, PlacePropsInterface } from 'spelieve-common/lib/Interfaces';
+
+import { PCT012MPlaceOneProvider } from '../../Models/PDB01MPlace/Contexts/PCT012MPlaceOne/MPlaceOne';
+import { PPA001Places } from '../PPA001Places';
+import { PPA002Place } from '../PPA002Place/Place';
+
+import { BottomTabParamList } from '@/App';
+import { PCT011MPlacesListProvider } from '@/Place/Models/PDB01MPlace/Contexts/PCT011MPlacesList';
+
+export type PlaceStackParamList = {
+	PPA001Places: PlacesPropsInterface; // 本来は PagePropsInterface を設定する
+	PPA002Place: PlacePropsInterface; // 本来は PagePropsInterface を設定する
+};
+
+const Stack = createNativeStackNavigator<PlaceStackParamList>();
+
+export const PlacePageNavigator = ({ navigation }: NativeStackScreenProps<BottomTabParamList, 'Place'>) => (
+	<PCT012MPlaceOneProvider parentDocRef={undefined} initialPlaceId="" language="ja">
+		<PCT011MPlacesListProvider>
+			<Stack.Navigator initialRouteName="PPA001Places">
+				<Stack.Screen
+					name="PPA001Places"
+					component={PPA001Places}
+					initialParams={{ country: '', administrativeAreaLevel1: '', administrativeAreaLevel2: '', locality: '' }}
```

Comment:
PageProps はundefined 許容にしてるから、↓みたいな、initialParamsになる想定だった！
```suggestion
					initialParams={{  }}
```

Reply:
[e4f155d](https://github.com/Ayato-kosaka/spelieve/pull/173/commits/e4f155d49528d4f521f6b35fc44781870446578d)で対応